ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -225,11 +225,13 @@ TFLiteInterpreter::invoke (const GstTensorMemory * input,
     status = kTfLiteError;
   }
 
+#if TFLITE_VERSION_MAJOR==1 && TFLITE_VERSION_MINOR==9
```

Comment:
there is another option like protobuf.
they manage the version like this 3.6.1 == 3006001.
if it looks more standardlike, please leave your opinion.

Reply:
We didn't check it, but let's say MINOR <= 9 here.

Reply:
need to check this with tf-lite 1.9.
IMO we dont need this.